hdl/testbench/xwb_fofb_shaper_filt/nvc/Manifest.py

```diff
@@ -0,0 +1,13 @@
+action = "simulation"
+sim_tool = "nvc"
+top_module = "xwb_fofb_shaper_filt_tb"
+target = "xilinx"
+syn_device = "xc7a200t"
+
+modules = {"local" : ["../"]}
+
+nvc_opt = "--std=2008"
+nvc_analysis_opt = "--relaxed"
+nvc_elab_opt = "--no-collapse"
+
+sim_post_cmd = "nvc -r %s --wave=%s.fst --format=fst"%(top_module, top_module)
```

Comment:
Use the `--dump-arrays` flag here to dump the signal arrays in the waveform file:

```
sim_post_cmd = "nvc -r --dump-arrays ...
```

Reply:
:+1: